setup/deb-setup.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+echo 'Setting up your developmental environment. This may take a while.'
+
+echo '[*] Adding keys...'
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update
+
+echo '[*] Installing GNUpg...'
+sudo apt-get install gnupg
+
+echo '[*] Adding Keys for rvm...'
+gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB && echo '[*] Installing rvm...' && \curl -sSL https://get.rvm.io | bash -s stable
+
+echo '[*] Installing Ruby-2.5.0...'
+sudo rvm install ruby-2.5.0
+
+echo '[*] Install node.js...'
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+
+echo '[*] Installing bundler...'
+gem install bundler
+
+echo '[*] Installing Gems and dependencies...'
+sudo apt-get install libmysqlclient-dev libpq-dev libqtwebkit-dev
+bundle install
+
+echo '[*] Installing yarn...'
+sudo apt-get install yarn
```

Comment:
This doesn't work without first adding a repo that has (the right) `yarn`. On debian, there may be a default package, `cmdtest`, which provides `yarn` and conflicts with the yarn deb repo.

Reply:
I shifted the command for adding the repo for yarn on the top so that for all the keys required we need to call `apt-get update` only once.